sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -6476,3 +6476,18 @@ def test_pretty_print_tensor_partial_deriv():
     ucode_str = ascii_str
     assert pretty(expr) == ascii_str
     assert upretty(expr) == ucode_str
+
+
+def test_issue_15560():
+    import sys
+    from sympy.core.compatibility import StringIO
+    fd = StringIO()
+    a = MatrixSymbol("a", 1, 1)
+    sso = sys.stdout
+    sys.stdout = fd
+    try:
+        pprint(a*(KroneckerProduct(a, a)), use_unicode=False, wrap_line=False)
+    finally:
+        sys.stdout = sso
```

Comment:
Write the test like the other tests in the file. There is no need to capture stdout.

Reply:
Updated the function ... 
Thanks